setup.py

```diff
@@ -50,4 +50,5 @@
         'Programming Language :: Python :: 3.7',
     ],
     namespace_packages=['sphinxcontrib'],
+    entry_points = {'console_scripts': 'oas2rst = sphinxcontrib.openapi.cmdline:main'}
```

Comment:
What is an use case for an external script? I mean who and under what conditions will be interested in producing an reStructuredText that heavily depends on `sphinxcontrib-httpdomain`?

Reply:
I agree with your comment. I wondered if it could be useful to someone else but since I needed it I decided to propose it.
I use it for two use cases:
- when the yaml file is not accessible from my Sphinx project. This happens in my company because of access right restriction difficult to bypass from automated builds
- when I want to adjust manually the content of the document generated, for instance to add a sample or a (Sphinx) link to another chapter.